src/applications/edu-benefits/feedback-tool/containers/IntroductionPage.jsx

```diff
@@ -30,6 +30,13 @@ class IntroductionPage extends React.Component {
           Get started right now by filling out a short form. Follow the steps
           below.
         </p>
+        <SaveInProgressIntro
+          messages={this.props.route.formConfig.savedFormMessages}
+          pageList={this.props.route.pageList}
+          downtime={this.props.route.formConfig.downtime}
+          prefillEnabled={this.props.route.formConfig.prefillEnabled}
```

Comment:
this control requires `prefillEnabled=true` to show the comprehensive signin message. otherwise it just shows a small signin link